plugins/Transitions/lang/en.json

```
@@ -1,42 +1,42 @@
 {
     "Transitions": {
         "BouncesInline": "%s bounces",
-        "DirectEntries": "Direct Entries",
+        "DirectEntries": "Direct entries",
         "Transitions": "Transitions",
         "ErrorBack": "Go back to the previous action",
         "ExitsInline": "%s exits",
         "NumPageviews": "%s pageviews",
         "NumDownloads": "%s downloads",
         "NumOutlinks": "%s outlinks",
         "TopX": "Top %s labels",
-        "FeatureDescription": "Transitions gives you a report that shows the things your visitors did directly before and after viewing a certain page. This page will explain how to access, understand, and use the powerful Transitions report.",
+        "FeatureDescription": "Transitions is a report showing the things your visitors did directly before and after viewing a given page. This page explains how to access, understand, and use the powerful \"Transitions\" report.",
         "AvailableInOtherReports": "Did you know? Transitions are also available as a row action in the following reports:",
-        "AvailableInOtherReports2": "Simply hover a row in any of these reports and click on the transition icon %s to launch it.",
-        "FromCampaigns": "From Campaigns",
-        "FromPreviousPages": "From Internal Pages",
+        "AvailableInOtherReports2": "Simply hover a row in any of these reports and click on the transition icon \"%s\" to launch it.",
+        "FromCampaigns": "From campaigns",
+        "FromPreviousPages": "From internal pages",
         "FromPreviousPagesInline": "%s from internal pages",
-        "FromPreviousSiteSearches": "From Internal Search",
+        "FromPreviousSiteSearches": "From internal search",
         "FromPreviousSiteSearchesInline": "%s from internal searches",
-        "FromSearchEngines": "From Search Engines",
-        "FromSocialNetworks": "From Social Networks",
-        "FromWebsites": "From Websites",
+        "FromSearchEngines": "From search engines",
+        "FromSocialNetworks": "From social networks",
+        "FromWebsites": "From websites",
         "IncomingTraffic": "Incoming traffic",
         "LoopsInline": "%s page reloads",
-        "NoDataForAction": "There's no data for %s",
-        "NoDataForActionDetails": "Either the action had no pageviews during the period %s or it is invalid.",
-        "PeriodNotAllowed": "Time period not allowed",
-        "PeriodNotAllowedDetails": "This feature is not available for this period, try selecting a period with fewer days",
+        "NoDataForAction": "No data for %s",
+        "NoDataForActionDetails": "Either the action had no pageviews during the %s period, or it is invalid.",
```

Just looked it up in the code, and it seems the placeholder is simply replaced with a `<br />`. I don't think that should be needed anymore, as the other error messages don't have one either.

```suggestion
        "NoDataForActionDetails": "Either the action had no pageviews during the period, or it is invalid.",
```